pkgs/os-specific/linux/kmscon/default.nix

```diff
@@ -9,19 +9,22 @@
 , pango
 , pixman
 , pkg-config
+, cmake
 , docbook_xsl
 , libxslt
+, mesa
+, ninja
 }:
 
 stdenv.mkDerivation rec {
   pname = "kmscon";
-  version = "unstable-2018-09-07";
+  version = "v9.0.0";
```

Comment:
```suggestion
  version = "9.0.0";
```

Reply:
Ah, good point, I should have checked that. Fixed.